### ISSUE TITLE
NERDTree detects Vim version incorrectly and doesn't load

### ISSUE THREAD

**Issue by vhalis** [scrooloose/nerdtree#992]
### Environment

* Operating System: Ubuntu 16.04
* Vim version `:version`: 
```
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Nov 24 2016 16:44:48)
Included patches: 1-1689
Extra patches: 8.0.0056
```

* NERDTree version `git rev-parse --short HEAD`: a3f0e66

### Process

1. Open Vim with NERDTree enabled

### Current Result

```
line   18:
NERDTree: this plugin requires vim >= 7.3. DOWNLOAD IT! You'll thank me later!
```

### Expected Result

NERDTree shouldn't complain about version

### Possible Fix

When I `:echo v:version` I get `704`, rather than something greater than `730` as the code expects.

```
if v:version < 730
  echoerr "NERDTree: this plugin requires vim >= 7.3. DOWNLOAD IT! You'll thank me later!"
  finish
endif
```
Perhaps that should be `< 703?`

**Comment by vhalis** [comment]
I have opened #994

**Comment by PhilRunninger** [comment]
Fixed by #994 (and #993)
